tests/benchmark/run-benchmarks.sh

```diff
@@ -60,4 +60,4 @@ sleep 2
     --address "${CLIENT_SOCK}" \
     --runtime.id "8000000000000000000000000000000000000000000000000000000000000000" \
     --benchmarks accounts_transfers \
-    --benchmarks.concurrency 5000
+    --benchmarks.concurrency 4000
```

Comment:
Had to reduce concurrency a bit, to make it work on CI.

Tested locally (with previous concurrency value):
- the results went from 850/s->450/s. I guess that's kind of expected since every transaction now also emits an event and account transfer does quite simple transactions